src/firefly/test/edu/caltech/ipac/firefly/server/query/lc/LightCurveProcessorTest.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+/**
+ * [April-23-19]
+ * Author: L.Z.
+ * This is an integration test since there is so many dependencies in LightCurveProcesor, thus, a unit test is not a right
+ * choice.
+ * Input data:
+ *   There are two types of input data, one is a table and the other is a URL.  The table input has two kinds, one is none
+ *   filtered table and the other is filtered table.   Both tables are  stored in firefly_test_data under the same package name.
+ *      212027909.tbl
+ *      212027909_filteredBJDLargeThan3270.tbl
+ *
+ *   The URL is taken from the original integration test written by Emmanuel:
+ *      url = "http://web.ipac.caltech.edu/staff/ejoliet/demo/AllWISE-MEP-m82-2targets-10arsecs.tbl";
+ *
+ * Expected data:
+ *   The expected data is stored in firefly_test_data under the same package name.
+ *     periodogramResult_212027909.voTbl
+ *     periodogramResult_212027909_filteredBJDLargeThan3270.voTbl
+ *
+ *    The expected data was created by running the api directly or through curl
+ *      curl -F upload=@/work/azhang_9027/pubspace/04/70/08a53fa0f79695e0f726aeef96f5/input.tbl \
+ *      -F alg=ls \
+ *      -F x=xCol \
+ *      -F y=yCol \
+ *      -F peaks=50 \
+ *       https://irsadev.ipac.caltech.edu:9028/cgi-bin/periodogram/nph-periodogram_api -o outputFile
```

Comment:
Try not to expose internal hosts in open-source firefly.  Use `irsa` instead.

Reply:
@loitly I cleaned up and changed from irsadev to irsa.  
Do I need to add  @Category({TestCategory.Integration.class}) to every @Test or just add one
at the beginning of the file?

Reply:
To the test(s) you want to denote as `integration`.  I don't know which one it is.

Reply:
LightCurveProcessorTest is an integration test.  There are two @Test method, I added  @Category({TestCategory.Integration.class}) before the @Test .  I was wondering if I could
just add it once at the beginning of the file instead of adding it to every test method.

Reply:
Oh.  I am not sure.  Try then let me know.